src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml

```diff
@@ -10,48 +10,42 @@
+        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/4.1.1/css/bootstrap.min.css"
               asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
         <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
     </environment>
 </head>
 <body>
-    <nav class="navbar navbar-inverse navbar-fixed-top">
-        <div class="container">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="sr-only">Toggle navigation</span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">Company.WebApplication1</a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
-                </ul>
+    <header>
+        <nav class="mb-3 navbar navbar-expand navbar-light flex-column flex-md-row bg-white border-bottom box-shadow">
+            <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index">Company.WebApplication1</a>
+            <ul class="navbar-nav ml-md-auto">
+                <li class="nav-item">
+                    <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Page1</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Page2</a>
+                </li>
+            </ul>
 @*#if (IndividualAuth || OrganizationalAuth)
-                <partial name="_LoginPartial" />
+            <partial name="_LoginPartial" />
 #elseif (WindowsAuth)
-                <p class="nav navbar-text navbar-right">Hello, @User.Identity.Name!</p>
+            <p class="nav navbar-text navbar-right">Hello, @User.Identity.Name!</p>
 #endif*@
-            </div>
-        </div>
-    </nav>
+        </nav>
+    </header>
 
-    <partial name="_CookieConsentPartial" />
-
-    <div class="container body-content">
-        @RenderBody()
-        <hr />
-        <footer>
-            <p>&copy; copyrightYear - Company.WebApplication1</p>
-        </footer>
+    <div class="container">
+        <partial name="_CookieConsentPartial" />
     </div>
+    <main role="main" class="container pb-1">
```

Comment:
Is it typical to have multiple elements using the `container` class on a single page? Especially one as simple as this?

Reply:
Damian, yes, the 'container' is an element to host the rows, that's the only role and it has no semantic meaning. The 'main' tag has one:
https://getbootstrap.com/docs/4.0/layout/grid/#how-it-works

Reply:
As I understand [this](https://getbootstrap.com/docs/3.3/css/#overview-container) there's nothing wrong with having multiple containers so long as they're not nested. It didn't take much effort to reduce it to one, so I did it anyway.